### PR TITLE
[author] [new] Add airbrake-js

### PR DESCRIPTION
**Pull request by zefer**
* https://github.com/airbrake/airbrake-js
* https://airbrake.io

JavaScript exception notifier library for [Airbrake.io](https://airbrake.io).

### CODE REVIEW

File: ajax/libs/airbrake-js-client/package.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  },
+  "homepage": "https://airbrake.io",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/airbrake/airbrake-js.git",
+    "basePath": "dist",
+    "files": [
+      "client.min.js",
+      "client.min.js.map",
+      "client.js"
```
Comment (PeterDaveHello) [inline ajax/libs/airbrake-js-client/package.json]: Maybe `client.*` will be even better.

File: ajax/libs/airbrake-js-client/package.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    "type": "git",
+    "url": "https://github.com/airbrake/airbrake-js.git"
+  },
+  "keywords": [
+    "exception",
+    "error",
+    "logging",
+    "airbrake",
+    "client"
+    ],
```
Comment (PeterDaveHello) [inline ajax/libs/airbrake-js-client/package.json]: Wrong indent here.

File: ajax/libs/airbrake-js-client/package.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  "author": "Airbrake",
+  "license": "MIT",
+  "bugs": {
```
Comment (PeterDaveHello) [inline ajax/libs/airbrake-js-client/package.json]: Please remove `bugs`

**Comment by zefer** [comment]
Thanks @PeterDaveHello. Updated.

File: ajax/libs/airbrake-js/package.json [hunk 4]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "airbrake-js",
+  "version": "0.5.6",
+  "description": "Notify Airbrake of JavaScript exceptions",
+  "main": "client.js",
```
Comment (maruilian11) [inline ajax/libs/airbrake-js/package.json]: hi, please also remove `main`, then i think the PR is ok.

Comment (zefer) [inline ajax/libs/airbrake-js/package.json]: Fixed. Thanks for the quick response @maruilian11!

**Comment by PeterDaveHello** [comment]
Merge now, @zefer if there is any update of structure of the `dist` folder, please also send a PR to us, so that we can grab the new versions.

**Comment by maruilian11** [comment]
:+1:

**Comment by zefer** [comment]
Thank you @PeterDaveHello @maruilian11.

**Comment by PeterDaveHello** [comment]
@zefer 👍

**Pull request merged**